src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -41,18 +42,42 @@ const ExplorationViewPage = ({
   const { windows, panel } = layout;
   const [selectedTab, setSelectedTab] = useState(panel);
   const [addMenuVisible, setAddMenuVisible] = useState(false);
-
+  const { method } = useSelector((state) => (
+    state.experimentSettings.processing?.configureEmbedding?.embeddingSettings || ' '
```

Comment:
Looking at the code, I don't see the need for adding this default of `' '`. You are checking everywhere if method is null.